Frontend/CO.CDP.OrganisationApp/CO.CDP.OrganisationApp.csproj

```diff
@@ -29,6 +29,6 @@
       <ProjectReference Include="..\..\Libraries\CO.CDP.Mvc.Validation\CO.CDP.Mvc.Validation.csproj" />
       <ProjectReference Include="..\..\Libraries\CO.CDP.Configuration\CO.CDP.Configuration.csproj" />
       <ProjectReference Include="..\..\Services\CO.CDP.Localization\CO.CDP.Localization.csproj" />
-    </ItemGroup>
+    </ItemGroup>  
```

Comment:
Unnecessary change.

Reply:
If the trailing spaces were added accidentaly, they should be removed to keep the PR clean and changes relevant. Next someone's in this file and their IDE reformats it, the change will be reverted and again cause an unrelated change on the PR.